docker-live/charts-participant/templates/mojaloop-simulator.yaml

```diff
@@ -19,6 +20,10 @@ spec:
       labels:
         app: {{ $participantId }}-mojaloop-simulator
     spec:
+      # volumes:
+      #   - name: jws-private-key
+      #     secret:
+      #       secretName: {{ $participantId }}-jws-private-key
       containers:
         - name: mojaloop-simulator
           image: mojaloop/mojaloop-simulator:v9.4.1.7-pisp
```

Comment:
please use: mojaloop/mojaloop-simulator:v9.4.1.8-pisp

Reply:
Yep, will do in the next PR